src/Models/IoTWorkspaceProject.ts

```diff
@@ -86,13 +73,26 @@ export class IoTWorkspaceProject extends IoTWorkbenchProjectBase {
           `Device root path ${this.deviceRootPath} does not exist.`);
     }
 
+    // Init and update iot workbench project file
     this.iotWorkbenchProjectFilePath =
         path.join(this.deviceRootPath, FileNames.iotWorkbenchProjectFileName);
-
-    // Update iot workbench project file.
     await updateProjectHostTypeConfig(
         scaffoldType, this.iotWorkbenchProjectFilePath, this.projectHostType);
 
+    // Init workspace config file

```

Comment:
I remember the same logic is shared by other place, you could extract as a method later

Reply:
Okay.